cloud/network-billing.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Network Billing
+description: This document outlines how network usage is charged in RisingWave Cloud.
+sidebarTitle: Network Billing
+---
+
+## Pricing Dimensions
+
+Network usage is billed based on the following dimensions:
+
+### 1. Public Network Traffic
+
+Network traffic charges are based on the volume of data moved across public internet, measured in gigabytes (GB). This includes:
+
+- **Ingress** (Data entering RisingWave):
+    - **Streaming Data Integration**: Continuously ingesting real-time data streams from external message brokers or sources into RisingWave for immediate processing.
+    - **Batch Data Imports**: Loading data from external systems, such as databases or cloud storage, into RisingWave for analytics or processing.
+- **Egress** (Data leaving RisingWave):
+    - **Data Export to External Destinations**: Sending processed data to external databases, message brokers, or other sinks.
+    - **Client Application Queries**: When client applications retrieve query results from RisingWave, the outbound data transfer constitutes egress traffic.
```

Comment:
In reality the traffic from queries are not counted since they are proxied via `rwproxy`, for both ingress and egress.

Either leaving the description or removing it is fine by me.